src/OrchardCore.Modules/Orchard.Media/Orchard.Media.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+  </PropertyGroup>
+  
```

Comment:
Use the following to have full razor intellisense.

    <Project Sdk="Microsoft.NET.Sdk.Web">

      <PropertyGroup>
        <TargetFramework>netstandard1.6</TargetFramework>
        <EnableDefaultContentItems>false</EnableDefaultContentItems>
        <OutputType>Library</OutputType>
      </PropertyGroup>
    ...